site/public/ts/digital/controllers/DigitalCopyController.ts

```diff
@@ -25,13 +27,32 @@ export class DigitalCopyController extends CopyController {
     }
 
     protected copy(e: ClipboardEvent, main: DigitalCircuitController): void {
+        console.log("HERE");
         const selections = main.getSelections();
         const objs = selections.filter((o) => o instanceof IOObject) as IOObject[];
 
         // Create sub-circuit with just selections to save
         const designer = new DigitalCircuitDesigner(-1);
         designer.addGroup(CopyGroup(objs));
 
+        const objs2 = designer.getObjects();
+        console.log(objs2);
+        for (const obj of objs2) {
+            if (isNode(obj)) {
+                console.log(obj);
+                console.log(obj.getConnections().length);
+                console.log(designer.getWires().length);
+                if ((obj.getConnections().length) < 2) {
+                    designer.removeObject(obj);
+                    if (designer.getWires().length == 1) {
```

Comment:
this is designer.getWires(), but don't we want to check obj.getConnections() still?
Cause designer.getWires().length == 1 is only true if there's 1 wire in the entire circuit